### ISSUE TITLE
Zoom to Place Name Widget

### ISSUE THREAD

**Issue by stdavis** [agrc/wfrc-wab-widgets#26]
Wrap Sherlock and point at a yet-to-be-created feature service that @msilski is working on.

**Comment by stdavis** [comment]
@msilski: Any progress on getting the feature service stood up for this?

**Comment by msilski** [comment]
Not yet. This is probably something I can get to in the next few weeks once I finish work on a couple products that are due for release in early May.

**Comment by stdavis** [comment]
No rush from my end. Just looking for an update. Thanks.

**Comment by stdavis** [comment]
Polygon feature service will be provided by WFRC.

**Comment by msilski** [comment]
I will finally be creating this feature service this week. Is there a preference if it is on ArcGIS Online or on WFRC's GIS server? I plan to add fields for name, scale, x, and y. Is there an ideal schema configuration for the service (field naming convention, field length, etc.)?

**Comment by stdavis** [comment]
No preference between AGOL and ArcGIS Server.

Field name doesn't matter as long as it's text. I don't believe that we support scale at the moment so I'll need to add some code to handle that. X & Y fields won't be necessary for me since I'll be looking at the geometry directly.

**Comment by msilski** [comment]
Here is the URL for the feature service:

https://gis.wfrc.org/arcgis/rest/services/General/ZoomToPlaceNames/FeatureServer/0

**Comment by msilski** [comment]
@stdavis Here is a screenshot of the CSS in a test application where Sherlock is implemented. The bootstrap appears to be overriding the font-family.
![CSS discrepancy](https://user-images.githubusercontent.com/1699870/62323502-a9340a00-b464-11e9-983d-bf87a7877c35.PNG)

**Comment by stdavis** [comment]
@msilski Try replacing this file in `widgets/Sherlock/css` and seeing if it fixes the issue. If it doesn't I'll commit the changes and it will be included in the next release.

[style.css.zip](https://github.com/agrc/wfrc-wab-widgets/files/3468784/style.css.zip)

**Comment by msilski** [comment]
I tested it and found that the font matched the original app except for a few small conflicts. I added a few CSS properties to that file that appear to have resolved those remaining differences. Here is my revised file:

[style.css.zip](https://github.com/agrc/wfrc-wab-widgets/files/3478333/style.css.zip)

The changes are specific to the RTP app. I imagine that my changes are not best practice and would better be done on a class level versus an element level, but hey, they work. You're welcome to commit the changes and include this file with the additional CSS properties in the next release.
